src/ecalc_neqsim_wrapper/thermo.py

```diff
@@ -446,17 +441,20 @@ def mix_neqsim_streams(
     mol_per_hour_1 = mass_rate_stream_1 / stream_1.molar_mass
     mol_per_hour_2 = mass_rate_stream_2 / stream_2.molar_mass
 
-    fraction_1 = mol_per_hour_1 / (mol_per_hour_1 + mol_per_hour_2)
-    fraction_2 = mol_per_hour_2 / (mol_per_hour_1 + mol_per_hour_2)
+    component_moles: dict[str, float] = defaultdict(float)
```

Comment:
```suggestion
    fluid_component_moles: dict[str, float] = defaultdict(float)
```

gir seg vel av kontekst, men er det "fluid component" det heter?